packages/apps/purchase-page/components/OfferPage/Stages/PaymentStage.js

```diff
@@ -192,6 +207,20 @@ class PaymentStage extends Component {
                 ReactGA.ga('ec:setAction', 'checkout', {
                   option: paymentType,
                 });
+
+                // Adding paypal purchase.
+                if (window && paymentType === 'PayPal') {
```

Comment:
This is going to throw on the server.

Use 
```suggestion
                if (typeof window !== 'undefined' && paymentType === 'PayPal') {
```